packages/frontend/stalker-app/src/app/modules/ports/list-ports/list-ports.component.ts

```diff
@@ -110,10 +112,25 @@ export class ListPortsComponent {
   );
 
   dataSource$ = this.ports$.pipe(
```

Comment:
Ça ressemble à quoi la performance mettons? Si on met le paging à 100, ça veut dire qu'on fait 100 calls on the spot? 👁️👃🏻👁️

Reply:
C'est backend maintenant, je sais pas trop, j'ai pas reussi a me debarasser du deep copy for some reason, si je le faisais pas, j'arrivais pas a set la propriete, meme apres un cast